### ISSUE TITLE
API Module - Initiated conversation resulting in internal server errors

### ISSUE THREAD

**Issue by Alrahambra** [freescout-helpdesk/freescout#983]
Hello,

Thank you for making the create conversation method to the API module! However it seems that something in the payload is causing the message sending to fail, resulting in error 500 on the server side or there is a bug that prevents sending of the message generally if I'm only attempting to send a single notification message for a customer, without existing message. 

However using the example payload in the API documentation fully works as expected and the messages leave the server successfully and it starts failing as soon as I remove the customer reply from the payload.

Here is the log line that I found to be generated:
```
Call to a member function getMainEmail() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getMainEmail() on null at /var/www/REDACTEDREDACTEDREDACTEDREDACTED/ticketing/Modules/ApiWebhooks/Http/Controllers/ApiController.php:454)

```
When I use the following payload, I've also tried to give it a customer as an email string and some other variations. Just in case I've also tried admins, users as senders and different customers to rule out errors:

```
{
	"type": "email",
	"mailboxId": 1,
	"subject": "API Test",
	"imported": false,
	"assignTo": 1,
	"status": "active",
	"customer": {
		"id": 1
	},
	"threads": [
		{
			"text": "This is the message from a user",
			"type": "message",
			"user": 1
		}
	]
}
```

**Comment by freescout-helpdesk** [comment]
What do you mean under "message sending"?

**Comment by Alrahambra** [comment]
> What do you mean under "message sending"?

I'm simply attempting to send a single message for a customer through the API without a note, or user reply.

**Comment by freescout-helpdesk** [comment]
Fixed in API & Webhooks Module v1.0.7
